de-de/einrichtung.md

```diff
@@ -0,0 +1,67 @@
+# Einrichtung des Modpacks
+
+**Magst du Video-Tutorials lieber als Text?** [**Klicke hier!**](https://github.com/Fabulously-Optimized/fabulously-optimized#reviews)
+
+### [Curseforge Launcher](https://download.curseforge.com)
+
+1. Gehe zu [Files](https://www.curseforge.com/minecraft/modpacks/fabulously-optimized/files)
+2. Klicke auf `Install` bei der ersten *Curseforge*-Version
+3. Akzeptiere den Download (falls gefragt wird)
+
+Oder, falls der Curseforge-Launcher schon offen ist:
+
+1. Wähle *Minecraft* von der Übersicht oder der Seitenleiste
+2. Suche "Fabulously Optimized"
+3. Drücke `Install`
+
+### [MultiMC](https://multimc.org)
+
+1.17 benötigt Java 16 oder höher, 1.18 hingegen Java 17 oder höher. [Offizielle Homepage](https://www.oracle.com/java/technologies/downloads/), [Adoptium](https://adoptium.net/download)
```

Comment:
Why promote Adoptium in particular? Should I refer the users to you when they are having problems with it?

Reply:
I had a lot of problems on Linux while trying to install the normal Java Runtime Environment. That's why I recommend Adoptium.
You can remove it of course, it's your repository.

Reply:
Okay. I actually did suggest Adoptium in the wiki at some point, but when I tested it with a non-technical user, it seemed to not create the JAVA_HOME variables on Windows, therefore it did not launch JAR files unfortunately.

Reply:
So, should I remove or keep the links?

Reply:
Remove, please.

Reply:
Ok.